pkgs/by-name/ma/mani/package.nix

```diff
@@ -1,9 +1,9 @@
 {
+  lib,
+  git,
```

Comment:
Using `gitMinimal` is perhaps a better option here?

Reply:
changed it, still works.